src/test/regress/expected/multi_basic_queries.out

```diff
@@ -22,6 +22,16 @@ SELECT avg(l_extendedprice) FROM lineitem;
  38141.835375000000
 (1 row)
 
+-- Verify that we can do queries in read-only mode
+BEGIN;
+SET TRANSACTION READ ONLY;
+SELECT count(*) FROM lineitem;
+ count 
+-------
+ 12000
+(1 row)
+
+COMMIT;
```

Comment:
Could we also add a COPY out in the read-only section, since that's so fundamental for pg_dump?